common/djangoapps/course_modes/models.py

```diff
@@ -608,6 +609,9 @@ def is_white_label(cls, course_id, modes_dict=None):
         if modes_dict is None:
             modes_dict = cls.modes_for_course_dict(course_id)
 
+        if configuration_helpers.is_site_configuration_enabled():
+            return configuration_helpers.get_value('show_ecommerce_reports', False)
```

Comment:
I think the issue is that all sites will (or should) have `is_site_configuration_enabled == True`, so this is almost a moot check but perhaps it is necessary.  Also I'm not sure we can guarantee that White Label courses will have a single PROFESSIONAL/NO_ID_PROFESSIONAL course mode, either now or in the future.

@douglashall mentioned yesterday that we might check the "org" of the site configuration to see if it is the "org" for the course.  If this is the case, then by edX rules we are working in a White Label course context.

I am wondering, however, if it would be enough to simply check the Site's "show_ecommerce_reports" setting.  If this flag is enabled then display the report buttons.  The only real question in my mind here is whether the setting should be at the Site level, and thus apply to all courses, or if it should be at the individual Course level (managed through Advanced Settings).  If we move it to the Course level then we can stop using the CourseMode.is_white_label() check for this feature altogether.